bin/compile

```diff
@@ -93,6 +93,11 @@ fi
 
 speak "Running stack"
 cd "$WORK_DIR"
+# Makefile support (if you need other build tools than Stack that you need to use in conjunction)
+if [ -f $BUILD_DIR/Makefile ]; then
+    speak "Makefile detected"
+    make -C $BUILD_DIR
+fi
```

Comment:
That's not the case - if different arguments are passed to stack in the Makefile, then the subsequent calls to stack build will undo them and replace the built binaries with new binaries. This is either/or behavior - if there's a Makefile present, it should control what gets built and installed. Leaving both the make and the naked stack calls undoes the expressed intention of the Makefile. The else is the default behavior, and the Makefile is overriding behavior.